backend/src/modules/Comment/CommentModule.js

```diff
@@ -1,5 +1,5 @@
 import { CommentController } from './controller/Comment.controller.js';
-import { AddCommentService } from './service/Comment.service.js';
+import { CommentService } from './service/Comment.service.js';
 import { InMemoryCommentRepository } from './repository/inMemory/inMemoryComment.repository.js';
```

Comment:
Nazwę tego pliku z dużej, bo eksportujesz klasę :)
```suggestion
import { InMemoryCommentRepository } from './repository/inMemory/InMemoryComment.repository.js';
```